test/test_server_go/EGILTestServer/main.go

```diff
@@ -115,7 +115,7 @@ func runTest(testName, testPath string,
 	testLogger *TestLogger,
```

Comment:
If we are merging this code in - its fine but we should turn this into a patch if this turns into standalone flake

Reply:
I thought I responded before resolving. Anyway, I created a couple of PRs for this and some other non-nix related thing, which are already merged. So after the rebase, these will not appear.